pages/quickstart/setting-up-a-local-environment-with-dive-cli.mdx

```diff
@@ -0,0 +1,165 @@
+import { Callout } from "nextra/components";
+
+# Setup Local Environment Using DIVE Cli
+
+## Introduction
+
+Welcome to the DIVE Command-Line Interface (CLI), This guide will help you get up and running with DIVE CLI, the all-in-one tool designed to simplify blockchain development, set up local environments, and enable seamless cross-chain communication.
+
+## Prerequisites
+
+Before diving into DIVE CLI, ensure you have the following prerequisites installed on your system:
+
+- [Docker installed and running](https://docs.kurtosis.com/install/#i-install--start-docker)
+- [Kurtosis installed and running](https://docs.kurtosis.com/install/#ii-install-the-cli) or [upgrading to the latest](https://docs.kurtosis.com/upgrade/)
+
+## Installation
+
+- Installing with Homebrew
+
+        ```sh
+        brew install hugobyte/tap/dive-cli
+        ```
+
+- Installing From Github [Releases](https://github.com/HugoByte/DIVE/releases)
+
+  ```sh
+  curl -L https://github.com/HugoByte/DIVE/releases/download/{version}/{releasefilename}.tar.gz > dive.tar.gz
+
+  tar -xzvf dive.tar.gz
+
+  mv dive /usr/bin/
+  ```
+
+## Setup Single Node
```

Comment:
Setting up a single node and then setting up a cross-chain connection fails (see https://github.com/HugoByte/DIVE/issues/77)

I think this section should be removed until that issue is fixed.